src/pages/saksbehandling/sakintro/KlageLister.tsx

```diff
@@ -102,19 +106,24 @@ const KlageLister = (props: { sakId: string; klager: Klage[]; vedtak: Vedtak[] }
                             );
                         }
 
-                        return (
-                            <LinkAsButton
-                                variant="secondary"
-                                size="small"
-                                href={Routes.klage.createURL({
-                                    sakId: props.sakId,
-                                    klageId: klage.id,
-                                    steg: hentSisteVurderteSteg(klage),
-                                })}
-                            >
-                                {formatMessage('klage.fortsettBehandling')}
-                            </LinkAsButton>
-                        );
+                        if (klage.avsluttet !== 'ER_AVSLUTTET') {
```

Comment:
De verdiene i `klage.avsluttet` borde vara i enums (hvis de er det så burde de brukes istedet for string-verdien här).